pages/Configuring/Animations.md

```diff
@@ -27,11 +21,19 @@ animation=windows,1,10,myepiccurve,slide
 `STYLE` (optional) is the animation style
 
 The animations are a tree. If an animation is unset, it will inherit its
-parent's values.
+parent's values. See [animation tree](#animation-tree).
```

Comment:
See *the* animation tree.

Reply:
Calm down english major. You *can* refer to a title without "the". Changed anyway.